packages/manager-react-components/src/components/order/Order.context.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useContext, useState } from 'react';
+
+export type TOrderContext = {
+  setIsOrderInitialized: (isOrderInitialized: boolean) => void;
+  isOrderInitialized: boolean;
+};
+
+const OrderContext = createContext<TOrderContext>({} as TOrderContext);
+
+export const OrderContextProvider: React.FC<React.PropsWithChildren> = ({
+  children,
+}) => {
```

Comment:
```suggestion
export const OrderContextProvider = ({
  children,
}: React.PropsWithChildren) => {
```

If we want uniformize